pkgs/build-support/node/fetch-yarn-deps/default.nix

```diff
@@ -148,23 +148,6 @@ in
             GIT_SSL_CAINFO = "${cacert}/etc/ssl/certs/ca-bundle.crt";
             NODE_EXTRA_CA_CERTS = "${cacert}/etc/ssl/certs/ca-bundle.crt";
 
-            supportedArchitectures = builtins.toJSON {
-              os = [
-                "darwin"
-                "linux"
-              ];
-              cpu = [
-                "arm"
-                "arm64"
-                "ia32"
-                "x64"
-              ];
-              libc = [
-                "glibc"
-                "musl"
-              ];
-            };
```

Comment:
Could you squash this `supportedArchitectures` json file commit to the one that added this?

Reply:
done